jumpscale/packages/admin/actors/admin.py

```diff
@@ -127,6 +127,7 @@ def get_developer_options(self) -> str:
         over_provision = j.core.config.set_default("OVER_PROVISIONING", False)
         explorer_logs = j.core.config.set_default("EXPLORER_LOGS", False)
         escalation_emails = j.core.config.set_default("ESCALATION_EMAILS_ENABLED", False)
+        auto_pool_extend = j.core.config.set_default("auto_extend_pool", False)
```

Comment:
the variable name used should be `auto_extend_pools` and the config param should `AUTO_EXTEND_POOLS_ENABLED`